Update bundle descriptions (#1274)
ezsystems/developer-documentation: docs/guide/bundles.md

You can also reuse the bundles you create in other projects or share them with the community.

Many [[= product_name =]] functionalities are provided through separate bundles included in the installation.
You can see the bundles that are automatically installed with [[= product_name =]]

> in the respective `composer.json` files, for example for [Ibexa Content](https://github.com/ibexa/content/blob/master/composer.json).

Reviewer: ```suggestion
in the respective `composer.json` files.
For example, for Ibexa Content, see the [JSON file on GitHub](https://github.com/ibexa/content/blob/master/composer.json).
```

To remove a bundle (either one you created yourself, or an out-of-the-box one that you do not need),
see the [How to Remove a Bundle](http://symfony.com/doc/5.0/bundles/remove.html) instruction in Symfony doc.

## Core packages

|Bundle|Description|
|---------|-----------|
|[ezsystems/ez-support-tools](https://github.com/ezsystems/ez-support-tools)|System information|
|[ezsystems/ezplatform-admin-ui-assets](https://github.com/ezsystems/ezplatform-admin-ui-assets)|Assets for the Back Office|
|[ezsystems/ezplatform-admin-ui](https://github.com/ezsystems/ezplatform-admin-ui)|Back Office interface|
|[ezsystems/ezplatform-content-forms](https://github.com/ezsystems/ezplatform-content-forms)|Form-based integration for the Symfony Forms into Content and User objects in kernel|
|[ezsystems/ezplatform-core](https://github.com/ezsystems/ezplatform-core)|Core system functionalities|

> |[ezsystems/ezplatform-cron](https://github.com/ezsystems/ezplatform-cron)|Cron package for use via a the `ezplatform:cron:run` command|

Reviewer: ```suggestion
|[ezsystems/ezplatform-cron](https://github.com/ezsystems/ezplatform-cron)|Cron package for use with the `ezplatform:cron:run` command|
```
